pyQuARC/code/custom_validator.py

```diff
@@ -96,7 +96,7 @@ def bounding_coordinate_logic_check(west, north, east, south):
         return result
 
     @staticmethod
-    def presence_check(*field_values):
+    def one_item_presence_check(*field_values):
```

Comment:
Should you not break the loop once you find a value? 

Merge all comments into doc strings.

Reply:
yes! good catch!